transfer-instructions.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: docs
+title: "Transfer Instructions"
+---
+
+## Transfer Overview & Setup
+
+An item must be inventoried in the Digital Media Log before it is transferred. See instructions for [Inventorying Digital Media Items](inventory). When a digital media item is imaged or otherwise has its contents transferred, this must be recorded in the Digital Media Log.
+
+## Destination Folders and File Names
+
+Newly created disk transfers are stored on Ginger; this is accessible via mounted shares on the FRED and BitCurator workstations. Please note that the Virus Checking Workstation is not backed up, so transfers should be scanned and moved to the Digital Records (X) drive as soon as possible. On Ginger and the X drive, all transfers are stored in a directory with the name of the ID auto-generated by the Digital Media Log. On the X drive, these directories are located in a parent directory with the FA or Accession number.
+
+## Choosing Transfer Method
+
+Disk images are single files containing the complete content and structure representing a data storage medium or device. There are different types of images. Forensic images perfectly replicate the structure and contents of a storage device, and allow the viewing of deleted files and slackspace (i.e, the unused space in a disk cluster). Logical images \[AD1\] are not true forensic images; and do not include file slackspace. At this point in time, disk images are created in-house for optical disks [(CDs and DVDs)](#cds-and-dvds), [external hard drives](#hard-drives-and-usb-flash-drives), [USB Flash drives](#hard-drives-and-usb-flash-drives), [3.5-inch floppy disks and 5.25-inch floppy disks](#3-and-1-half-inch-and-5-and-1-quarter-inch-floppy-disks) (recently dated disks that are PC-formatted). The hardware used to create images will vary depending on the type of digital media.
```

Comment:
This paragraph could use some editing for clarity and content. I'm not sure what it's saying.